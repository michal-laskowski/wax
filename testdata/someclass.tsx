export class SomeClass { 
    private toReturn : any
    
    constructor(x) {
      this.toReturn = x ?? "- derfault -"
    }

    getValue(){
        return this.toReturn
    }

}


export const done = "a is done";