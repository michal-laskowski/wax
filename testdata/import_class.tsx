
import * as AllImports from "./someclass"

import { SomeClass as SingleImport } from "./someclass.tsx"

export default function View() {
    const fromAll = new AllImports.SomeClass("from-all")
    const fromSingle = new SingleImport("SingleImport")
 
    return <>
        <case-1>{ fromAll.getValue() }</case-1>
        <case-2>{ fromSingle.getValue() }</case-2>
    </>
}
