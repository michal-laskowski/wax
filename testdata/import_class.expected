<case-1>from-all</case-1>
<case-2>SingleImport</case-2>